gradle/libs.versions.toml

```diff
@@ -0,0 +1,44 @@
+[versions]
+hypertrace-serviceFramework = "0.1.43"
+hypertrace-grpcUtils = "0.9.0"
+hypertrace-documentStore = "0.7.18"
+
+protoc = "3.21.2"
+grpc = "1.48.0"
+mockito = "4.8.1"
+jackson = "2.13.4.2"
```

Comment:
nit: this isn't a full jackson version - jackson hotfixes are inconsistent across modules (i.e. most jackson modules don't have a 2.13.4.2, only a 2.13.4). So it works since we only use databind, but probably just want to inline this version there to avoid a mistake in the future (or rename it specifically jacksonDatabind would be fine too)